app/views/shared/_message_summary.html.haml

```diff
@@ -0,0 +1,14 @@
+- summarizer = MessageSummarizer.new(controller)
+- if summarizer.messages?
+  %li
+    %ul.nav.pull-right
+      %li.dropdown.pull-right
+        = link_to "Notices (#{summarizer.message_count})",
+          :notifications,
```

Comment:
Should this actually be linking to anything? The notifications link will show up in the dropdown, correct?